unison-src/transcripts/idempotent/duplicate-names.md

```diff
@@ -58,7 +58,7 @@ structural ability X where
 ``` ucm :added-by-ucm
   Loading changes detected in scratch.u.
 
-  I found two types called X:
```

Comment:
This change is due to the fact that our parser has two different error constructors that represent the duplicate type names. This PR performed only minimal unification (switching one over to the other), but there's a little bit more cleanup to do, namely delete the other one entirely.